polling/exchangeratesapi.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) of parts are held by the various contributors (see the CLA)
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+package polling
+
+import (
+	"encoding/json"
+	"github.com/cenkalti/backoff"
+	"github.com/zpatrick/go-config"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+)
+
+type ExchangeRatesAPIResponse struct {
+	Date    string
+	Base    string
+	Rates   ExchangeRatesAPIRecord
+}
+
+type ExchangeRatesAPIRecord struct {
```

Comment:
Can we change this to a map of floats? This just seems klugy.
See what I mean in this playground.
https://play.golang.org/p/Ynmo2ZSC4fj

Then we can write the list of currencies we want, and do it all dynamically.